### ISSUE TITLE
Return proper HTTP status for invalid content type

### ISSUE THREAD

**Issue by zevaryx** [lus/pasty#42]
Making a request to `/api/v2/pastes` fails with `invalid character 'x' looking for beginning of value`. 

Code to reproduce:
```py
import requests
resp = requests.post("https://pasty.lus.pm/api/v2/pastes", data={"content": "Test paste"})
```
![image](https://user-images.githubusercontent.com/57541873/155200673-32305d79-1980-49bc-b6e1-1958bea1da6a.png)

**Comment by lus** [comment]
I looked into the documentation of the `requests.post` method and it seems like the `data` parameter is being form encoded which is not supported by pasty (however, it shouldn't return a 500, this seems like an issue on my side that I have to fix). Please try to use the `json` parameter instead.

**Comment by zevaryx** [comment]
Yep, doing `requests.post("...", json=...)` solved it. Thanks for the quick feedback! I'll leave this open since there's some work needed with it
